page-optimize.php

```diff
@@ -121,7 +124,8 @@ function page_optimize_should_concat_css() {
 		return $_GET['concat-css'] !== '0';
 	}
 
-	return !! get_option( 'page_optimize-css', page_optimize_css_default() );
+	$should_concat = !! get_option( 'page_optimize-css', page_optimize_css_default() );
+	return apply_filters( 'page_optimize_should_concat_css', $should_concat );
```

Comment:
While we're revisiting conditional loading, we might as well support a first-class should-concat filter for CSS.